extension/src/popup/components/sendPayment/SendTo/index.tsx

```diff
@@ -32,18 +32,42 @@ import {
 
 import "../styles.scss";
 
+export const AccountDoesntExistWarning = () => (
+  <div className="SendTo__info-block">
+    <InfoBlock className="SendTo__info-block">
+      The destination account doesn't exist. Send at least 1 XLM to create
```

Comment:
```suggestion
      The destination account doesn’t exist. Send at least 1 XLM to create
```
Super NIT: but we try to use smart apostrophes/quotes in text strings so they're not the same as a code '

I'm kind of bad about this, too, so call me out on it if you see me do it!
https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#text